packages/common/src/utils/id.ts

```diff
@@ -4,3 +4,9 @@ import cuid from 'cuid';
 export { cuid };
 
 export const objectID = (): string => new ObjectId().toHexString();
+
+export const OBJECT_ID_REGEX = /"([\dA-Fa-f]{24})"/g;
+
+// please feel free to suggest more performant implementations
```

Comment:
i can come up with a better implementation tomorrow

Reply:
Do you want to deeply check all `string` properties of the object and replace them? That would work too.
Keep in mind we also have to replace the property name as well.

Reply:
yes, i have an idea on how to implement that behavior in a pretty succinct way